rioxarray/rioxarray.py

```diff
@@ -1078,3 +1079,90 @@ def transform_bounds(self, dst_crs, densify_pts=21, recalc=False):
         return rasterio.warp.transform_bounds(
             self.crs, dst_crs, *self.bounds(recalc=recalc), densify_pts=densify_pts
         )
+
+    def write_gcps(self, gcps, grid_mapping_name=None, inplace=False):
```

Comment:
What are your thoughts on adding a `crs` argument instead of passing in a tuple?

My current thoughts are that it would make it more clear to the user what input is expected, would remove the need for `gcps, gcp_crs = gcps` below, and would produce a clearer error message if they didn't provide the `crs` and instead provided a list of `gcps`.

Additionally, it would be more consistent with the input to the `rio.reproject` function.

Reply:
Good point, I'll fix this in the morning.

Reply:
Done